frontend/packages/ceph-storage-plugin/src/components/ocs-install/internal-mode/install-wizard-steps/select-capacity-and-nodes-step.tsx

```diff
@@ -42,6 +59,110 @@ const validate = (scName, enableMinimal, enableFlexibleScaling): ValidationType[
   return validations;
 };
 
+const isArbiterDisabled = (nodesData: NodeKind[], validateSelectedNodes?: boolean) => {
+  const validation = {
+    nodesLen: validateSelectedNodes ? 4 : 5,
+    zoneLen: validateSelectedNodes ? 2 : 3,
```

Comment:
```suggestion
    nodesLength: validateSelectedNodes ? 4 : 5,
    zoneLength: validateSelectedNodes ? 2 : 3,
```
Let's just go with the full names.